flavors/standard-EXASOL-all-r-4.4/flavor_base/build_deps/packages/apt_get_packages

```diff
@@ -2,7 +2,7 @@ coreutils|8.32-4.1ubuntu1.2
 locales|2.35-0ubuntu3.9
 tar|1.34+dfsg-1ubuntu0.1.22.04.2
 curl|7.81.0-1ubuntu1.20
-openjdk-11-jdk|11.0.26+4-1ubuntu1~22.04
+openjdk-11-jdk|11.0.27+6~us1-0ubuntu1~22.04
```

Comment:
Not a headless, needs a ticket

Reply:
created https://github.com/exasol/script-languages-release/issues/1132